docs/administrator-handbook.md

```diff
@@ -191,6 +191,16 @@ both direct HTTP URLs as well as local files:
 ```
 $ rpm-ostree override replace https://kojipkgs.fedoraproject.org//packages/podman/3.3.1/1.fc34/x86_64/podman-3.3.1-1.fc34.x86_64.rpm
 ```
+```
+$ curl https://rpmfind.net/linux/fedora/linux/updates/testing/38/Everything/x86_64/Packages/p/podman-4.5.1-1.fc38.x86_64.rpm --output podman.rpm
+$ rpm-ostree override replace ./podman.rpm
```

Comment:
This needs admin to run, maybe can update like $ sudo rpm-ostree override replace ./podman.rpm ? And the next 2 commands.